frontends/web/src/routes/settings/settings.tsx

```diff
@@ -339,8 +351,12 @@ class Settings extends Component<Props, State> {
                                                             <div className="m-top-quarter m-bottom-quarter">
                                                                 <p className="m-none">{t('settings.expert.splitAccounts')}</p>
                                                                 <p className="m-none">
-                                                                    <Badge type="primary">BB02</Badge>
-                                                                    <Badge type="secondary" className="m-left-quarter">BB02-BTC</Badge>
+                                                                    <Badge type="generic">BB02</Badge>
+                                                                    <span className="text-gray">(</span>
```

Comment:
![image](https://user-images.githubusercontent.com/1225267/87294341-e7a2c780-c503-11ea-8653-fde94c9ae158.png)

Without a space it looks kind of strange to me, and with a space it also doesn't work... maybe a `-` before the parens?

Reply:
I want to change it to `BitBox02 (Multi, Bitcoin-only)` ok?